pandas/tests/io/parser/test_common.py

```diff
@@ -1904,12 +1905,21 @@ def test_suppress_error_output(all_parsers, capsys):
     assert captured.err == ""
 
 
+def __should_skip_utf8_test():
```

Comment:
be more specific here on the name of the skip

Reply:
Renamed to `__windows_ansi_encoding_not_cp1252`